docs/recipes/WritingTests.md

```diff
@@ -258,46 +258,38 @@ describe('todos reducer', () => {
 
 A nice thing about React components is that they are usually small and only rely on their props. That makes them easy to test.
 
-First, we will install [Enzyme](http://airbnb.io/enzyme/). Enzyme uses the [React Test Utilities](https://facebook.github.io/react/docs/test-utils.html) underneath, but is more convenient, readable, and powerful.
+First, we will install [React Testing Library](https://testing-library.com/docs/react-testing-library/intro). React Testing Library is built on top of React-DOM and React-DOM/test-utils, but is more convenient, readable, and powerful.
 
 ```sh
-npm install --save-dev enzyme
+npm install --save-dev @testing-library/react
 ```
 
-We will also need to install Enzyme adapter for our version of React. Enzyme has adapters that provide compatibility with `React 16.x`, `React 15.x`, `React 0.14.x` and `React 0.13.x`. If you are using React 16 you can run:
-
-```sh
-npm install --save-dev enzyme-adapter-react-16
-```
-
-To test the components we make a `setup()` helper that passes the stubbed callbacks as props and renders the component with [shallow rendering](http://airbnb.io/enzyme/docs/api/shallow.html). This lets individual tests assert on whether the callbacks were called when expected.
+To test the components, we `render` them into the DOM and pass stubbed callbacks as props, then we assert wheter the callbacks were called when expected.
 
 #### Example
```

Comment:
This example is not a connected component. It doesn't have anything to do with redux, so I'm a bit puzzled why it's here.

One other thing I'd note about this page is the section on "Connected Components" should be changed to *not* recommend exporting the unconnected component, and instead recommending utilizing the `wrapper` option for React Testing Library's `render` function. And to take it further, people should make their own render method that does this automatically: https://testing-library.com/docs/react-testing-library/setup#custom-render

Reply:
The only problem with using `wrapper` is that AFAIK it should be a comopnent, meaning I can't give it an `initialState` for example since it doesn't pass any props to `React.createElement` but it's children.. Am I missing something? It means that I'd probably wouldn't want to place it in a setup file. 
The `wrapper` option is a good option for other providers like shown in the example exactly, but for redux, I'm not sure it's so useful to what people would want to test..
The example in https://testing-library.com/docs/example-react-redux is more suitable and I think it can fit well in the docs here. WDYT?